docs/source/01_introduction/01_introduction.md

```diff
@@ -1,8 +1,8 @@
 # What is Kedro?
 
-Kedro is an open-source Python framework that applies software engineering best-practice to data and machine-learning pipelines.  You can use it, for example, to optimise the process of taking a machine learning model into a production environment. You can use Kedro to organise a single user project running on a local environment, or collaborate in a team on an enterprise-level project.
+Kedro is an open-source Python framework that applies software engineering best-practices to data and machine-learning pipelines.  You can use it, for example, to optimise the process of taking a machine learning model into a production environment. You can use Kedro to organise a single-user project running on a local environment, or collaborate in a team on an enterprise-level project.
```

Comment:
Should best-practices be hyphenated? I know it was already, but I actually think it shouldn't be.

FWIW: https://books.google.com/ngrams/graph?year_end=2019&year_start=1800&case_insensitive=on&smoothing=3&corpus=26&content=best+practice%2Cbest+practices%2Cbest-practice%2Cbest-practices&direct_url=t4%3B%2Cbest%20practice%3B%2Cc0%3B%2Cs0%3B%3Bbest%20practice%3B%2Cc0%3B%3BBest%20Practice%3B%2Cc0%3B%3BBest%20practice%3B%2Cc0%3B%3BBEST%20PRACTICE%3B%2Cc0%3B.t4%3B%2Cbest%20practices%3B%2Cc0%3B%2Cs0%3B%3Bbest%20practices%3B%2Cc0%3B%3BBest%20Practices%3B%2Cc0%3B%3BBest%20practices%3B%2Cc0%3B%3BBEST%20PRACTICES%3B%2Cc0%3B.t4%3B%2Cbest%20-%20practice%3B%2Cc0%3B%2Cs0%3B%3Bbest%20-%20practice%3B%2Cc0%3B%3BBest%20-%20practice%3B%2Cc0%3B%3BBest%20-%20Practice%3B%2Cc0%3B.t4%3B%2Cbest%20-%20practices%3B%2Cc0%3B%2Cs0%3B%3Bbest%20-%20practices%3B%2Cc0%3B%3BBest%20-%20Practices%3B%2Cc0%3B%3BBest%20-%20practices%3B%2Cc0#t4%3B%2Cbest%20practice%3B%2Cc0%3B%2Cs0%3B%3Bbest%20practice%3B%2Cc0%3B%3BBest%20Practice%3B%2Cc0%3B%3BBest%20practice%3B%2Cc0%3B%3BBEST%20PRACTICE%3B%2Cc0%3B.t4%3B%2Cbest%20practices%3B%2Cc0%3B%2Cs0%3B%3Bbest%20practices%3B%2Cc0%3B%3BBest%20Practices%3B%2Cc0%3B%3BBest%20practices%3B%2Cc0%3B%3BBEST%20PRACTICES%3B%2Cc0%3B.t4%3B%2Cbest%20-%20practice%3B%2Cc0%3B%2Cs0%3B%3Bbest%20-%20practice%3B%2Cc0%3B%3BBest%20-%20practice%3B%2Cc0%3B%3BBest%20-%20Practice%3B%2Cc0%3B.t4%3B%2Cbest%20-%20practices%3B%2Cc0%3B%2Cs0%3B%3Bbest%20-%20practices%3B%2Cc0%3B%3BBest%20-%20Practices%3B%2Cc0%3B%3BBest%20-%20practices%3B%2Cc0

Reply:
wow 🤯 
this is such a nice way to look into a word's spelling, because it's data driven. First time I see this, and going to bookmark this